crates/hulk/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![recursion_limit = "256"]
```

Comment:
Is this needed?

Reply:
Yes, compilation fails with very ugly error messages otherwise.

Reply:
Can we somehow fix the root cause instead of silencing the symptom?

Reply:
The error occurs when the compiler tries to parse the serialize_hierarchy macro output on the generated main outputs type of the control crate.
I don't think we should optimize for this recursion limit.